src/shared/constants.ts

```diff
@@ -2,6 +2,10 @@ import { ChainID } from '@stacks/transactions';
 
 export const gaiaUrl = 'https://hub.blockstack.org';
 
+export const FULL_PAGE_MAX_WIDTH = 440;
```

Comment:
Maybe we could move the constants related to layouts in its own file.

Reply:
Ok, yep, that makes sense. 👍